packages/CB-serverless-backend/api/order/createOrder.js

```diff
@@ -62,15 +63,22 @@ export const main = (event, context, callback) => {
 				'orderDate': new Date().toISOString()
 			}			
 			return createAndSaveOrder(completeOrder)
-		})
+    })
+    .then(() => {
+      // Batch updates the sold and unsold quantities after placing an order
+      return batchUpdateAvailableAndSoldQuantities(cartItems);
```

Comment:
Instead of sending `cartItems`, we can send `idToGroceryDataMapping` so that you can remove that `find` logic in `batchUpdateAvailableAndSoldQuantities` function.
You can get `qty` of a grocery in the card by direct object key look up.